_data/pages/navigation.yml

```diff
@@ -0,0 +1,125 @@
+primary:
+  - name: Features
+    url: /pages/features/
+  - name: Success Stories
+    url: /pages/success-stories/
+  - name: Documentation
+    url: /pages/documentation/
+  - name: Pricing
+    url: /pages/pricing/
+  - name: Knowledge Base
+    url: /pages/knowledge-base/
+  - name: Contact
+    url: /pages/contact/
+    # Look inside assets/js/index.js to see that how this url changes depending on enviroment
+secondary:
+  - name: Platform Status
+    url: https://cloudgov.statuspage.io/
+  - name: Login
+    url: https://pages.cloud.gov
+    external: true
+    class: usa-button usa-button-secondary nav-button
+
+footer:
+  - text: Home
+    href: /pages/
+  - text: Features
+    href: /pages/features/
+  - text: Success Stories
+    href: /pages/success-stories/
+  - text: Documentation
+    href: /pages/documentation/
+  - text: Pricing
+    href: /pages/pricing/
+  - text: Knowledge Base
+    href: /pages/knowledge-base/
+  - text: Contact
+    href: /pages/contact/
+  - text: Manage Sites
+    href: https://pages.cloud.gov
+    external: true
+
+sidenav:
```

Comment:
is there no way to do this dynamically? this won't be up to date as soon as someone creates a new page